lib/charms/opensearch/v0/helper_security.py

```diff
@@ -117,3 +123,38 @@ def to_pkcs8(private_key: str, password: Optional[str] = None) -> str:
     finally:
         os.unlink(tmp_key.name)
         os.unlink(tmp_pkcs8_key.name)
+
+
+def run_cmd(command: str, args: str = None) -> SimpleNamespace:
```

Comment:
i think this can be moved to another module, as it's unrelated to `*_security`. Feel free to create a new module for it if needed.

Reply:
has been moved to helper_charm.py